CoreTesting/nbproject/genfiles.properties

```diff
@@ -1,8 +1,8 @@
 build.xml.data.CRC32=5c21f638
 build.xml.script.CRC32=b5f9ba5a
-build.xml.stylesheet.CRC32=15ca8a54@2.79
+build.xml.stylesheet.CRC32=15ca8a54@2.80
```

Comment:
Should this file be checked in?

Reply:
There aren't any genfiles in the head of Constellation so I am going with no.  I will remove this from the checkin PR and add it to the git ignore.